src/udx.c

```diff
@@ -1965,6 +1965,7 @@ udx_init (uv_loop_t *loop, udx_t *udx) {
   udx->packets_rx = 0;
   udx->packets_tx = 0;
 
+  udx->packets_dropped_by_kernel = -1;
```

Comment:
what does -1 mean?

Reply:
that it's not supported, since it uses the Linux-specific SO_RXQ_OVFL socket option.